app/packs/src/apps/mydb/elements/details/samples/SampleDetails.js

```diff
@@ -243,14 +245,15 @@ export default class SampleDetails extends React.Component {
       .then((result) => {
         if (!result || result == null) {
           NotificationActions.add({
-            title: 'Error on Sample creation', message: `Cannot create molecule with this smiles! [${smi}]`, level: 'error', position: 'tc'
+            title: 'Error on Sample creation', message: `Cannot create molecule with entered smiles/Cas! [${smi}]`, level: 'error', position: 'tc'
```

Comment:
Another minor spelling remark: would be nice to spell "smiles" as "Smiles", and "Cas" as "CAS".